packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -2781,7 +2781,7 @@ class SemanticsConfiguration {
   ///
   /// This setting is often used in combination with [isSemanticBoundary] to
   /// create semantic boundaries that are either writable or not for children.
-  bool explicitChildNodes = false;
+  bool? explicitChildNodes = false;
```

Comment:
If this is not nullable, then almost every test in rendering/reattach_test.dart fails

Reply:
This is surprising... running this test locally to take a closer look.

Reply:
Instead of doing this, we should just default `explicitChildNodes` to false here: 

https://github.com/flutter/flutter/blob/606abc915fb85e6896c85e19c1cf3fa4f526745a/packages/flutter/lib/src/rendering/proxy_box.dart#L3624-L3628

Just like we do for the `excludeSemantics` flag. Not defaulting it to false seems like an oversight.

Reply:
done!